src/main/java/com/ledikom/utils/BotResponses.java

```diff
@@ -73,6 +76,11 @@ public static String helloCoupon(final int helloCouponDiscount) {
         return "Приветственный купон -" + helloCouponDiscount + "% на вашу следующую покупку.";
     }
 
+
+    public static String newCoupon(final String description, LocalDateTime expirationDate) {
+        return description + "\n\nАкция действует до " + expirationDate.format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));
```

Comment:
Construct a date string by yourself, we want it to be in dd.MM.yyyy HH:mm format (no such formatters in java as I know)